lib/src/transformer/transformer.dart

```diff
@@ -68,8 +81,9 @@ class _TransformAssetWriter implements AssetWriter {
   _TransformAssetWriter(this.transform);
 
   @override
-  Future writeAsString(build.Asset asset, {Encoding encoding: UTF8}) async =>
-      transform.addOutput(_toBarbackAsset(asset));
+  Future writeAsString(build.Asset asset, {Encoding encoding: UTF8}) async {
```

Comment:
Personally I like just using `async`, because it is less to type/scan. Unless you think there is a significant performance impact?